### PR TITLE
Projecttt Ganttt Charttt updattte.

### PR DESCRIPTION
**Pull request by hjoliver**
This is our Gantt Chart update from the final day in Exeter.

Any one reviewer will do!

### CODE REVIEW

**Comment by hjoliver** [comment]
(Conflicts with @sadielbartholomew's PR just merged!)

**Comment by hjoliver** [comment]
(Rebased and deconflicted).

File: docs/cylc-8-tasks.md [hunk 1]
```diff
@@ -31,74 +31,82 @@ ISODATETIME LIBRARY             *JAN. FEB. MAR. APR. MAY. JUN. JUL. AUG. SEP. OC
...
-  WS UI Rosie Go                |    |    |    |    |    |    |    |    |    |    |    |    |????| \      |
+  rosie disco Tornado           |   .|....|....|...x|    |    |    |    |    |    |    |    |    | \      |
+  Python 3                      |   .|....|....|....|...x|    |    |    |    |    |    |    |    | /      |
+  Setup.py, packaging, PyPi     |    |    |    |....| .. | .. |.o  |    |    |    |    |    |    | \      |
+  WS UI Rose Config-Edit        |    |    |    |    |    |    |    | ?  |    |    |    |    |    | \      |
+  WS UI Rosie Go                |    |    |    |    |    |    |    | ?  |    |    |    |    |    | \      |
 o-------------------------------o-----------------------------------------------------------o-------------o
 PRODUCTION (Cylc-8 + Rose-2)    *JAN. FEB. MAR. APR. MAY. JUN. JUL. AUG. SEP. OCT. NOV. DEC.*JAN.| /  SEP.  
-  Testing and Refinement        |    |    |    |    |    |    |    |    |    |    |    |    |....|.\. ...o|
+  Testing and Refinement        |    |    |    |    |    |    |    |    |    |    |    |    |....|.\. ...o| (hopefully June)
```
Comment (kinow) [inline docs/cylc-8-tasks.md]: Fingers crossed!

Comment (hjoliver) [reply docs/cylc-8-tasks.md]: :crossed_fingers:

**Pull request merged**